internal/api/front/perses/src/app/project/prometheusrule/prometheusrule-list/prometheusrule-list.component.html

```diff
@@ -50,27 +69,29 @@ <h2>{{gr.name}}</h2>
                 </tr>
               </table>
               <div class="map-container">
-                <div class="map" *ngIf="rule.labels">
+                <div class="map labels-map" *ngIf="rule.labels">
                   <h3>Labels</h3>
                   <table>
-                    <tr *ngFor="let label of rule.labels| keyvalue">
+                    <tr *ngFor="let label of rule.labels | keyvalue">
                       <th>{{label.key}}</th>
                       <td>{{label.value}}</td>
                     </tr>
                   </table>
                 </div>
-                <div class="map" *ngIf="rule.annotations">
+                <div class="map annotations-map" *ngIf="rule.annotations">
                   <h3>Annotations</h3>
                   <table>
-                    <tr *ngFor="let annotation of rule.annotations| keyvalue">
+                    <tr *ngFor="let annotation of rule.annotations | keyvalue">
                       <th>{{annotation.key}}</th>
                       <td>{{annotation.value}}</td>
                     </tr>
                   </table>
                 </div>
               </div>
-              <mat-divider *ngIf="!isLastGroup || !isLastRule"></mat-divider>
-            </mat-card-content>
+              <div class="divider">
+                <mat-divider *ngIf="!isLastRule"></mat-divider>
```

Comment:
mmm that's weird. The condition `!isLastGroup` is not necessary anymore ?

Reply:
nope we don't put the divider only if the alert is the latest of the group. The alert can be the latest of the group or the latest of the latest group, it does not change anything.

I think it was coming from the fact that there was a bug, the group was duplicated for every alert